test/save-in-progress/01-sip-review.e2e.spec.js

```diff
@@ -33,17 +33,18 @@ module.exports = E2eHelpers.createE2eTest(
     // server error when saving in progress form
     client
       .url(reviewUrl)
-      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible('.schemaform-sip-save-link', Timeouts.normal)
       .mockData({
         path: '/v0/in_progress_forms/1010ez',
         verb: 'put',
         value: {},
         status: 500
       }, token)
-      .click('.schemaform-sip-save-link');
+      .pause(500)
```

Comment:
Hmm... :thinking: Do we need to pause if we're already waiting for it to be visible?

Reply:
Same as here: https://github.com/department-of-veterans-affairs/vets-website/pull/7248#discussion_r167944540